x/wasm/keeper/bench_test.go

```diff
@@ -100,3 +102,26 @@ func BenchmarkCompilation(b *testing.B) {
 		})
 	}
 }
+
+// Calculate the time it takes to prune about 100k wasm codes
+func BenchmarkPruningWasmCodes(b *testing.B) {
```

Comment:
Good start to get some numbers.
This runs with a mem db, tiny contracts and no duplicate instances. I can imagine disk IO can have some impact here but let's add another DB to compare them